r2/r2/templates/link.html

```diff
@@ -102,12 +102,11 @@
         <div style="display: none;" id='lastViewed' time="${ epochtime(clicked._date if clicked else None) }"></div>
       %endif
       ${parent.midcol(thing.votable)}
-      %if thing.comments_enabled:
-        <%
-          num_comments = 0 if not thing.num_comments else thing.num_comments
-        %>
-        <a class="comment" href="${thing.url}#comments">${_('Comments (%d)') % (num_comments)}</a>
-      %endif
```

Comment:
Just double-checking that you're sure you want to remove the link to the comments here?